packages/application/style/menus.css

```diff
@@ -31,7 +31,8 @@
 }
 
 .lm-MenuBar-item {
-  padding: 0 8px;
+  /* stylelint-disable-next-line csstree/validator */
+  padding: 0 min(calc(1.17vw - 2px), 8px);
```

Comment:
Hi @gabalafou the **calc** was added to this part of code because from the previous PR (https://github.com/jupyterlab/jupyterlab/pull/14626) @krassowski commented that the before changes meant the the UI was cutting off the menu on 100% zoom. So **calc** was added to fix this issue.  
However if you think this is not needed then then it can be taken out.

Reply:
If what this line does, specifically the `- 2px` part, cannot be explained, then I think it should be simplified. Or there should be some comment that explains what process was used to come up with these values.

At the moment, I really can't understand why the `calc` expression is needed. If you do `min(1.2vw, 8px)`, then the screen width has to drop to 666px before anything changes—because 0.012 * 666 < 8.

Here is a screenshot showing that there's nothing wrong with the menu bar when I load this PR via GitPod and use Chrome Dev Tools to change the padding value to `0 min(1.2vw, 8px)`:

<img width="1022" alt="" src="https://github.com/jupyterlab/jupyterlab/assets/317883/d24c5b7e-0ab0-4d8d-a82d-6d6ad6f328d2">

Reply:
Hi @gabalafou I have made the changes yo have recommend about taking out the **calc** from the padding. Please have a look, at the recent commit for the changes.